upgrade/breaking-changes/11-0-0.md

```diff
@@ -10,6 +10,26 @@ position: 7
 
 This article lists the breaking changes introduced with the Telerik UI for .NET MAUI 11.0.0 release version.
 
+## Common Changes
+
+The `SkiaSharp` is updated to `3.116.1` version.
```

Comment:
* The SkiaSharp what? API? Integration? Tool? Platform?
* to version `3.116.1`; monospace formatting for the version is not something that I would use; however, if it's a common practice across the MAUI documentation, stick with it for consistency.